frontend/src/component/common/PermissionIconButton/PermissionIconButton.tsx

```diff
@@ -33,26 +36,42 @@ interface ILinkProps extends IPermissionIconButtonProps {
     to: string;
 }
 
-const PermissionIconButton = ({
+const RootPermissionIconButton = (props: IButtonProps | ILinkProps) => {
+    const access = useHasRootAccess(
```

Comment:
root access check without change request checking because we don't have project and env available